src/uPlot.js

```diff
@@ -1485,14 +1485,18 @@ export default function uPlot(opts, data, then) {
 		return sc.min + pct * d;
 	}
 
+	function yScaleValueAtPos(pos, scale) {
+		return scaleValueAtPos(plotHgtCss - pos, scale);
```

Comment:
since `scaleValueAtPos()` already contains an internal check for `xScaleKey` to auto-set the `dim`. why not just move the automatic `pos` inversion into there as well, instead of making a dedicated `yScaleValueAtPos()`.